jetty-server/src/main/java/org/eclipse/jetty/server/AsyncContentProducer.java

```diff
@@ -30,6 +30,14 @@
 class AsyncContentProducer implements ContentProducer
 {
     private static final Logger LOG = LoggerFactory.getLogger(AsyncContentProducer.class);
+    private static final Throwable UNCONSUMED_CONTENT_EXCEPTION = new IOException("Unconsumed content")
+    {
+        @Override
+        public synchronized Throwable fillInStackTrace()
```

Comment:
What is the reason for the `synchronized` here?

Reply:
It is part of the signature of the `super` implementation, so it was kept by accident. Since it's useless, it got removed in a subsequent revision.

Reply:
This is the current `jetty-10.0.x` of the same UNCONSUMED_CONTENT_EXCEPTION 

https://github.com/eclipse/jetty.project/blob/jetty-10.0.x/jetty-server/src/main/java/org/eclipse/jetty/server/AsyncContentProducer.java#L33-L40

https://github.com/eclipse/jetty.project/blob/b139cbfc7bafd23689b24a406290bec2fa14070d/jetty-server/src/main/java/org/eclipse/jetty/server/AsyncContentProducer.java#L33-L40

Jetty releases since 10.0.4 has the code block without the synchronized modifier.